websites.json

```
@@ -30,5 +30,13 @@
     "url": "https://robinroses.xyz",
     "rss": "https://robinroses.xyz/feed.xml",
     "owner": "robin@isabelroses.com"
+  },
+  {
+    "name": "isabel roses",
+    "slug": "isabelroses",
+    "about": "i'm a nix obssessive who occationally writes in real coding langauages",

```

Typo in `occationally` - should be `occasionally`?


```suggestion
    "about": "i'm a nix obssessive who occasionally writes in real coding languages",
```


(Sorry, GitHub is my main social network nowadays >.<')

Also, _hi, nice to meet you_

---

hi. spelling is hard ;o;